### PR TITLE
Fixed #1149

### LINKED ISSUE #1149: <CardList/> styles are broken
**Describe the bug**

**To Reproduce**

Meetup list on ethereum.org/community/

Current
Image 2020-06-25 at 4 53 27 PM

**Expected behavior**

Should match previous implementation:
Image 2020-06-25 at 4 53 53 PM

**Screenshots**

**Desktop (please complete the following information):**
All

**Smartphone (please complete the following information):**
All

**Additional context**
Regression from #1134

### PR DESCRIPTION
**Pull request by ryancreatescopy**
Fixed styling issues on the community meet-up card list to restore it to it's pre-Gatsby look-and-feel.

## Related Issue

#1149 

## Screenshots (if appropriate):

image

### CODE REVIEW

File: src/components/MeetupList.js [hunk 1]
```diff
@@ -246,6 +246,8 @@ const Table = styled.div`
 `
 
 const Item = styled(Link)`
+  flex-direction: row;
```
Comment (samajammin) [inline src/components/MeetupList.js:249]: FYI, by default any element that is `display: flex` is also `flex-direction: row`, so this is not necessary.

**Comment by samajammin** [comment]
This looks great @ryancreatescopy! I added a couple minor changes & will merge.

FYI, I don't think the mobile styles are strong:
Image 2020-07-06 at 5 37 55 PM
 
But they're still an improvement vs. current mobile styles:
Image 2020-07-06 at 5 38 09 PM

So I'm going to merge these as-is. Perhaps as a future improvement we break out the location text on a new line.

**Comment by samajammin** [comment]
@all-contributors please add @ryancreatescopy for code.

**Comment by ryancreatescopy** [comment]
Agreed, ideally on mobile that description area wraps to sit under the meetup name. This becomes more problematic for the more general component where they may already be two lines of copy. Let's discuss!

**Pull request merged**